community/hacktoberfest_2022.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hacktoberfest 2022 at Redis
+linkTitle: Hacktoberfest 2022
+description: Let's collaborate this October!
+weight: 1
+---
+
+[Hacktoberfest](https://hacktoberfest.com/) is a month-long online festival which takes place every year in October. It is sponsored by [DigitalOcean](https://www.digitalocean.com/) and aims to encourage people to get involved in open source projects.
+
+## How to get involved
+Through October, we're creating a number of GitHub issues for folks who want to contribute to our documentation and demo apps. [View our list of current open issues](https://github.com/search?l=&q=%23hacktoberfest+user%3Aredislabs-training+user%3Aredis+user%3Aredis-developer+user%3ANodeRedis+label%3Ahacktoberfest&state=open&type=Issues).
```

Comment:
I was able to get the link to produce issues by removing the #hacktoberfest hash tag: https://github.com/search?q=user%3Aredislabs-training+user%3Aredis+user%3Aredis-developer+user%3ANodeRedis+label%3Ahacktoberfest&type=Issues

Do we want to update this?